kernel/data-pipeline/cdc/client/src/main/java/org/apache/shardingsphere/data/pipeline/cdc/client/importer/ImporterFactory.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.cdc.client.importer;
+
+/**
+ * Importer factory.
+ */
+public final class ImporterFactory {
+    
+    /**
+     * Get importer.
+     *
+     * @param databaseType database type
+     * @return importer
+     */
+    public static Importer getImporter(final String databaseType) {
+        switch (databaseType) {
+            case "openGauss":
+                return new OpenGaussImporter();
+            default:
+        }
+        return null;
+    }
```

Comment:
It's better to use SPI for better extension by user (since user could not modify `ImporterFactory`), TODO could be added for now